lib/jxl/encode_test.cc

```diff
@@ -1338,11 +1340,20 @@ TEST(EncodeTest, JXL_BOXES_TEST(BoxTest)) {
         FAIL();  // unexpected status
       }
     }
+
     EXPECT_EQ(0, memcmp(exif_data, dec_exif_box.data(), exif_size));
     EXPECT_EQ(0, memcmp(xml_data, dec_xml_box.data(), xml_size));
-  }
 }
 
+std::string nameBoxTest(
+    const ::testing::TestParamInfo<std::tuple<bool>>& info) {
+    return (std::get<0>(info.param) ? "C" : "Unc") + std::string("ompressed");
+}
+
+INSTANTIATE_TEST_SUITE_P(EncodeBoxParamsTest, EncodeBoxTest,
+                         ::testing::Combine(::testing::Values(false, true)),
```

Comment:
of course it is a bit silly to use a 1-tuple `std::tuple<bool>>`, we are planning to make this a two-tuple soon....

Reply:
making it a two-tuple, because apparently older version of gtest don't support `Combine()` with only one argument, causing the debian test to fail...